schemas/measuredQuantity.schema.tpl.json

```diff
@@ -0,0 +1,4 @@
+{
+    "_type": "https://openminds.ebrains.eu/controlledTerms/MeasuredQuantity",
```

Comment:
could we add also "local field potential" etc here? or would you keep those separate for example under a terminology named "neuronal signal" ?

Reply:
In the schema we've developed here, "local field potential" would be a recording, not a measurement, but maybe the distinction isn't important.

It occurs to me we could perhaps add this information to the Channel schema.

Reply:
measuredQuantity sounds like a general terminology. why would you differentiate recording from it? for me a recording is also a measurement, no? But I understand if you want to differentiate between extracellular measurements and intracellular measurements OR do you want to differentiate between scalar values and timeseries?

Reply:
the latter - we've chosen to represent a measurement at a single point in time as a Measurement (scalar - stored directly in KG), a sequence of measurements as a Recording (timeseries - stored in a File)

Reply:
but I agree it doesn't make sense to distinguish MeasuredQuantity and RecordedQuantity, so we should use MeasuredQuantity for both Measurement and Recording/Channel.

Reply:
okay